docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst

```diff
@@ -17,7 +17,7 @@ To add a storage location:
       :width: 100%
    .. tip::
 
```

Comment:
The /location_management/ section showed some discrepancies in the UI. Corrected the text to suit the current UI. No pictures yet.